aten/src/ATen/native/native_functions.yaml

```diff
@@ -845,6 +845,10 @@
 
 - func: full_like(Tensor self, Scalar fill_value, *, ScalarType dtype, Layout layout, Device device, bool pin_memory=False) -> Tensor
 
+- func: from_file(str filename, bool? shared=None, int? size=0, *, ScalarType? dtype=None, Layout? layout=None, Device? device=None, bool? pin_memory=None) -> Tensor
```

Comment:
Why do we need `pin_memory` to be an optional bool? Isn't the default simply `false`?

Reply:
Because our codegen folds TensorOptions only if they all specified similarly.

Reply:
In that case why did we miss the tracer functions prior to that patch?